v_frame/src/plane.rs

```diff
@@ -424,7 +424,8 @@ impl<T: Pixel> Plane<T> {
     }
   }
 
-  pub fn downsampled(
```

Comment:
would be better to keep the old name to not cause an hard API break.

Reply:
:+1: